src/SpectatorView.Unity/Assets/SpectatorView/Scripts/StateSynchronization/NetworkedComponents/AudioSource/AudioSourceBroadcaster.cs

```diff
@@ -21,6 +21,8 @@ public enum ChangeType
             Properties = 0x8
         }
 
+        public override string PerformanceComponentName => "AudioSourceBroadcaster";

```

Comment:
>"AudioSourceBroadcaster" [](start = 59, length = 24)

Is there a reason we can't use .GetType().Name for these? If not, it would be nice to use nameof(AudioSourceBroadcaster) so that renaming the component will automatically update these references too. Same goes for all of these overrides.

Reply:
nameof required a singleton definition. I will try GetType().Name